bson/bson_corpus_spec_test.go

```diff
@@ -350,6 +350,9 @@ func runTest(t *testing.T, file string) {
 					case "0x00", "0x05", "0x13":
 						var doc D
 						err := UnmarshalExtJSON([]byte(s), true, &doc)
+						if strings.Contains(p.Description, "Null") {
```

Comment:
Could you leave a comment here (for posterity) explaining why "Null" test-cases need to be `Marshal`ed to raise the correct error?